eZ/Publish/Core/Limitation/ContentTypeLimitationType.php

```diff
@@ -174,4 +191,46 @@ public function valueSchema()
     {
         throw new \eZ\Publish\API\Repository\Exceptions\NotImplementedException(__METHOD__);
     }
+
+    /**
+     * Evaluate permissions to create new Version.
+     *
+     * @param \eZ\Publish\SPI\Limitation\Target\Version $version
+     * @param \eZ\Publish\API\Repository\Values\User\Limitation $value
+     *
+     * @return bool|null
+     */
+    private function evaluateVersionTarget(
+        Target\Version $version,
+        APILimitationValue $value
+    ): ?bool {
+        $accessVote = self::ACCESS_ABSTAIN;
+
+        // ... unless there's a specific list of target translations
+        if (!empty($version->allContentTypeIdsList)) {
+            $accessVote = $this->evaluateMatchingAnyLimitation(
+                $version->allContentTypeIdsList,
+                $value->limitationValues
```

Comment:
@mikadamczyk are these values always string, the one coming from Limitations, not from the new property?